src/python/pants/notes/master.rst

```diff
@@ -4,6 +4,68 @@ Master Pre-Releases
 This document describes ``dev`` releases which occur weekly from master, and which do
 not undergo the vetting associated with ``stable`` releases.
 
+1.3.0dev3 (12/02/2016)
+----------------------
+
+A weekly unstable release.
+
+API Changes
+~~~~~~~~~~~
+
+* Bump setuptools version. (#4103)
+  `Pull Request #4103 <https://github.com/pantsbuild/pants/pull/4103>`_
```

Comment:
wonder if we shouldn't abbreviate these as "PR" or just the "#number" as standard parlance? e.g.

`PR #4103`

or

`#4103`

or even better - use the `(#4103)` in the title as the anchor for the PR link and kill the default sub-bullet altogether?